### DISCUSSION TITLE
INCOMING REWARDS NOT ARRIVED IN WALLET

### DISCUSSION THREAD

**Discussion by stlanzel** [Chia-Network/chia-blockchain#7070] [Support]
CHIA shows me 2 XCH was farmed but after several days the wallet remains empty (only 9 mojo).
wallet and full node are synced.

what is the reason?
is there any way to fix the problem and make available the XCH in the wallet?
In any case I would like to be sure the problem is fixed and next time no any XCH will be lost.

![Immagine](https://user-images.githubusercontent.com/84446400/124022845-a3972400-d9ed-11eb-9915-f236a4dd2781.png)

thanks in advance for coop

**Comment by loppefaaret** [comment]
this could be a potential comprimised wallet, where an attacker is moving out funds from it!
please try and give the full receive address of the wallet, the ones listed in the "To" column

**Reply by stlanzel** [reply]
you can find below the three incoming:
Incoming
xch1rrp0d3cyfpc2kth43cm788dcae2e8ykk8g4993fyhw6tfjexrjsszsh0y4
06/23/2021 13:38:25
Confirmed at height 472322
0,000000000009

Incoming
xch1wkugp0l67q7uc0mxs2np9zdqqx8axnmlkunr7dy8p5dw8p04pccqkuy2gd
06/23/2021 04:07:17
Confirmed at height 470474
0,250000000009


Incoming
xch1wkugp0l67q7uc0mxs2np9zdqqx8axnmlkunr7dy8p5dw8p04pccqkuy2gd
06/23/2021 04:07:17
Confirmed at height 470474
1,75

**Comment by joelrb** [comment]
Check your config.yaml. What is set in the payout address "xch_target_address"? If the same wallet in GUI then check Chia explorer and look where your xch went. If wallet in config is different, then you may need to look for the keys (mnemonic).

**Reply by stlanzel** [reply]
in the config.yaml I can currently see:
xch_target_address: xch1wkugp0l67q7uc0mxs2np9zdqqx8axnmlkunr7dy8p5dw8p04pccqkuy2gd

**Reply by stlanzel** [reply]
CHIA EXPLORER shows following info regarding xch1wkugp0l67q7uc0mxs2np9zdqqx8axnmlkunr7dy8p5dw8p04pccqkuy2gd.
could you please help me to understand them:

![image](https://user-images.githubusercontent.com/84446400/124106626-a84aef00-da64-11eb-9ec3-03a8b3d62a9b.png)

**Reply by joelrb** [reply]
Oh no. The status is spent. Click that green link and it will show you the status. It wen't somewhere that's why your wallet is zero.

You hacked? What software did you install? HPool? Others?

**Reply by stlanzel** [reply]
I really do not know if I am hacked. no pool, but in the past i used some chia software tools.....

**Reply by loppefaaret** [reply]
if you did not send 2 XCH, you **are** hacked, and need to do something about it!

**Reply by stlanzel** [reply]
I have to delete also all my old plot (1500) or I can continue to farm them with the new chia installation (if possible)?

**Reply by loppefaaret** [reply]
keep the plots!
and keep the old key!
but generate a new key from a clean system!
then take the receive address from the wallet of the new key, and transfer the last 0.0000009 XCH to it, and also - and this is the most important, input it as your "Farming Reward Target Address" in the GUI, as described below
this way, you can still use your old compromised wallet key, to farm your plots, but have the rewards send directly into a new wallet!

**Comment by loppefaaret** [comment]
https://xchscan.com/address/xch1wkugp0l67q7uc0mxs2np9zdqqx8axnmlkunr7dy8p5dw8p04pccqkuy2gd
the coins a sendt to another address - if this is not you that have done this, you private key is comromised - and if you cannot identify how it got compromised, the only suggestion i can give you, is to completely format and reinstall your machine, and reinstall chia, generate a new wallet, and start using this new wallets receive address as you "Farming Reward Target Address", which is set in the GUI under the "Farm" tab, in the top right, under the 3 `...` dots - set both addresses to this new address from the newly generated wallet - and be more careful of what software you run on your machine, and who have access to it... 
https://github.com/Chia-Network/chia-blockchain/wiki/Chia-Keys-Management

**Comment by jack60612** [comment]
Can someone close this?

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
